users/templates/users/profile_page.html

```diff
@@ -5,45 +5,45 @@
   <link rel="stylesheet" href="{% static 'users/css/profile_page.css' %}">
 {% endblock %}
 {% block body_content %}
-  <div class="profilepage">
```

Comment:
Merge conflict with develop that has to be resolved. You wrote this to begin with @tOgg1 so you would be better off resolving this.